cupy/cuda/cublas.pyx

```diff
@@ -320,23 +320,23 @@ cpdef dger(size_t handle, int m, int n, double alpha, size_t x, int incx,
 # BLAS Level 3
 ###############################################################################
 
-cpdef sgemm(size_t handle, int transa, int transb,
```

Comment:
I think this interface should use `int`, because `cublasSgemm` needs `int`.
How about fixing caller code.